.github/workflows/pr_metrics.yml

```diff
@@ -0,0 +1,59 @@
+name: Post-Sprint PR metrics
+on:
+  workflow_dispatch:
```

Comment:
isn't this missing something? I don't see a value of `workflow_dispatch`

Reply:
I don't think it needs one? cf. https://github.com/marketplace/actions/issue-metrics